micronutrientsupport/micronutrient-support-tool
# feat: update cost effectiveness route

- [x] resolves issue 1172
- [x] cost effectivess removed from quicklmaps and  routing established
- [x] reformatted intevrntion ID copy/email section

To test:

1. Navigate to Cost Effectiveness, via Quickmaps OR map tools.
2. Create an intervention / load a recently created one
3. Go through form, all pages should load / function.
4.( Cost Summary is failing to load data, this is an issue prevalent in the develop branch too )
